Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:bionic
+
+#Locale
+ENV LANG C.UTF-8
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV GOLANG_VERSION="1.16.2"
+
+ENV NODE_VERSION="15"
+
+# Install needed packages and clean up.
+RUN apt-get update \
+	&& apt-get install -y curl wget git make gcc g++ ca-certificates --no-install-recommends \
```

Comment:
put apt arguments before pacakge names, then sort packages one per line in A-Z order